runtime/src/bank.rs

```diff
@@ -871,7 +869,15 @@ impl Bank {
         if parent.epoch() < new.epoch() {
             new.apply_feature_activations(false);
         }
+        let cloned = new
+            .stakes
+            .read()
+            .unwrap()
+            .clone_with_epoch(epoch, new.stake_program_v2_enabled());
```

Comment:
Actual `clone()`-ing moved here because I wanted to touch runtime feature (via `.stake_program_v2_enabled()`).

Reply:
There is same change in #13461